### PR TITLE
Magical Item Room (2nd PR)

### PR DESCRIPTION
**Pull request by carlopault**
image

## Changelog
:cl: Carlospaul
added: Chapel now has a Magical Item Storage. 
added: New helmet: The Afro Maker (Assisted by Yoyobatty)
added: New shoes: Ratvar's Boots of Deceleration (Assisted by Protozoa)
/:cl:

### Why It's Good For the Game
Chaplain should have more magic items available to him besides a null rod, flask of holy ass, and holy book. 

### Items in the room
- Chaplain's null rod
- Flask of holy water
- mysterious old shard (1 use soul shard) 
- Afro Maker
- Ratvar's Boots of Deceleration

### CODE REVIEW

**Comment by Zyzarda** [comment]
+1 more fun

**Comment by ExpAntique** [comment]
+1

**Comment by mkikt4743** [comment]
yeah

**Comment by cris5311** [comment]
+1
fun

**Comment by byandaccount** [comment]
+1 just for the one-use soulstone that might be used on a benomorph one day.

**Comment by MrEpicSax** [comment]
+1 yeee

**Comment by Maroonenahmoron** [comment]
+1, but make the shard a holy converted one (blue) so people don't go screaming **CULT** if they find a shade wondering around the halls.

**Comment by carlopault** [comment]
You want me to make it purified? Oh boy time for coder nonsense

**Comment by Punkinguy** [comment]
+1 sounds fun but should have even more variation

**Comment by carlopault** [comment]
> +1 sounds fun but should have even more variation

I can only bitch and moan to Yoyobatty/Protozoa so much to help me with coding before they tell me to learn myself.

**Review by AnthonyJNgo — APPROVED** [review]
+3 looks good Carlos if this breaks anything I will hunt you down.

File: hippiestation/code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -11,4 +11,15 @@
...
 	alternate_worn_icon = 'hippiestation/icons/mob/head.dmi'
-	alternate_screams = list('hippiestation/sound/voice/stop_right_there.ogg', 'hippiestation/sound/voice/stop_right_there2.ogg')
\ No newline at end of file
+	alternate_screams = list('hippiestation/sound/voice/stop_right_there.ogg', 'hippiestation/sound/voice/stop_right_there2.ogg')
+
+/obj/item/clothing/head/helmet/redtaghelm/afromaker
+	name = "Afro Maker"
+	desc = "To wear this helmet is to accept its creator."
+	color = "#FF0000"
+	resistance_flags = INDESTRUCTIBLE
```
Comment (yoyobatty) [inline hippiestation/code/modules/clothing/head/helmet.dm:20]: ugh literally why didn't you use a fucking space

**Pull request merged**